packages/contracts-periphery/contracts/universal/citizen-house/IBadge.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
Interface contracts should always be permissive in their version so that other projects can import them